packages/core/js-peer/src/connection/index.ts

```diff
@@ -134,6 +134,12 @@ export class RelayConnection extends FluenceConnection {
 
     async connect(onIncomingParticle: ParticleHandler) {
         await this._lib2p2Peer.start();
+        
+        // TODO: make it configurable
```

Comment:
Le'ts mention this task in TODO description: [DXJ-328](https://linear.app/fluence/issue/DXJ-328/make-max-number-of-streams-configurable)